### DISCUSSION TITLE
App ID invalid for ReactJS

### DISCUSSION THREAD

**Discussion by AJoParker** [speechly/speechly#213] [Q&A] [answered]
Hello, this is my first-time use using Speechly. I have followed the instructions for using Speechly with the react-speech-recognition package for ReactJS. However, I used the app ID shown on the project's dashboard; it keeps saying the app ID is invalid.

**Comment by aukkonen** [comment, marked as answer]
@HollowCrusader 
There are a couple of things that could be the issue here.

1) Did you get the id from a page that says "Project settings" at the top? If yes, this is the "project id", but you need an App ID. To get an App ID you should first create a new application. Its App ID is shown in the top right corner in the apps configuration screen.

2) If you got the App ID from the right place, you should also remember to *deploy* the application by clicking the Deploy button located on the right side of the apps configuration screen.

Hope this helps!

**Reply by AJoParker** [reply]
I was having an issue with #2. As it turned out, the first application wouldn't deploy, no matter what I did. So, I created the second application and got it deployed with no problem.
